nncf/openvino/graph/node_utils.py

```diff
@@ -95,7 +85,7 @@ def get_bias_value(node_with_bias: NNCFNode, nncf_graph: NNCFGraph, model: ov.Mo
     add_node = nncf_graph.get_next_nodes(node_with_bias)[0]
     bias_constant = get_node_with_bias_value(add_node, nncf_graph)
     ov_bias_constant = ops_dict[bias_constant.node_name]
-    return get_const_value(ov_bias_constant)
+    return ov_bias_constant.data
```

Comment:
Why don't keep the `get_const_value` with the `return const_node.data`?

Reply:
I've just thought that the function only with `return const_op.data` is redundant, and it is simpler to just use `const_op.data`. However, if you have certain concerns about this, I may revert it and keep the `get_const_value()` function.

Reply:
Personally, I prefer to wrap such things that are used in several places (eg. everywhere) into the method.
But it needs to have opinions from all reviewers to decide. I do not insist on reverting.

Reply:
@alexsu52 Could you please provide your opinion about this?

Reply:
I've updated code according to comments.